src/components/Footer/Footer.css

```diff
@@ -0,0 +1,67 @@
+.footer {
+  display: flex;
+  width: 100%;
+  background-color: #faf9ec;
```

Comment:
This can also be added with tailwind using the config file that I pushed (it wasn't merged yet, but when it does, this can be done with tailwind)